src/fping.c

```diff
@@ -1323,6 +1332,15 @@ void finish()
     if (stats_flag)
         print_global_stats();
 
+    if (reachable_flag) {
+        if ((num_hosts-num_unreachable >= reachable)) {
+            printf(">=%d hosts are reachable\n", reachable);
+        } else {
+            printf("<%d hosts are reachable\n", reachable);
+        }
+        exit(0);
```

Comment:
I think that for this to be useful it should return a different exit code, depending on failure or success (and hopefully scripts using this won't rely on the output...)

Reply:
I added different exit codes, 0 for success (if >= N hosts are reachable), and 1 for failure (if < N hosts are reachable).